.github/workflows/deploy-to-aks-feast.yaml

```
@@ -20,7 +21,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

suggestion: We can now update to `v5`, which was released [a few weeks ago](https://github.com/actions/checkout/releases/tag/v5.0.0).
```suggestion
        uses: actions/checkout@v5
```